.env.example

```diff
@@ -44,6 +44,11 @@ AWS_DAILY_EXPORT_ACCESS_KEY_ID=<key_id>
 AWS_DAILY_EXPORT_SECRET_ACCESS_KEY=<secret_key>
 AWS_DAILY_EXPORT_BUCKET=<bucket_name>
 
+AWS_BOXI_EXPORTS_BUCKET=<bucket_name>
+AWS_DAILY_EXPORT_ACCESS_KEY_ID=<key_id>
```

Comment:
```suggestion
AWS_BOXI_EXPORT_ACCESS_KEY_ID=<key_id>
```

Reply:
Although having made this suggestion – what are we using this env var for? Can't see it in use but maybe I've missed something obvious!

Reply:
No, you are not. I have missed adding that file in. Is the config for AWS gem!